src/components/ListDetailPanel/ListDetailPanel.tsx

```diff
@@ -36,14 +47,15 @@ export const ListDetailPanel = ({
   ariaLabelledBy,
   children,
   className,
+  completed,
   id,
   variant,
   title,
   ...other
 }: Props) => {
   const componentClassName = clsx(styles['list-detail__panel'], className, {});
   return (
-    <div
+    <nav
```

Comment:
Why is this element now a nav? I can see why the `tablist` would be a `nav` element but not the `tabpanel`.

Reply:
This will be revisited in the a11y iteration so I probably should have just left it alone for now. Reverting to <div>.

Reply:
Actually, looking at it again, I think you reverted the other `nav` but left this one.

Reply:
This one is still a `nav` element btw.